package/MDAnalysis/analysis/rms.py

```diff
@@ -576,22 +577,26 @@ def run(self, start=None, stop=None, step=None, progout=10, quiet=False):
         progout : int (optional)
             number of frames to iterate through between updates to progress
             output; ``None`` for no updates [10]
-        quiet : bool (optional)
-            if ``True``, suppress all output (implies *progout* = ``None``)
-            [``False``]
+        verbose : bool (optional)
+            if ``False``, suppress all output (implies *progout* = ``None``)
+            [``True``]
```

Comment:
I kept the default to be consistent with the rest of the module that had them explicitly stated in the docstring. Also, the default in the signature (`None`) does not match the effective default (`True`).

Removing all the explicit defaults from docstring should be a separate issue as they are everywhere in the library.